### ISSUE TITLE
Hey can you please add profiles like in Netflix please?

### ISSUE THREAD

**Issue by Noamrko** [Stremio/stremio-features#448]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by TRtomasz** [comment]
closing as currently not possible

**Comment by guybrush-threepwood-the-third** [comment]
Out of curiosity, could you summarize why this isn't feasible? Issues with the current data model of users that make adding profiles difficult?

**Comment by TRtomasz** [comment]
https://github.com/Stremio/stremio-features/issues/622#issuecomment-4673832722
